frontend/packages/console-metal3/src/models.ts

```diff
@@ -0,0 +1,15 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const BaremetalHostModel: K8sKind = {
+  label: 'Bare Metal Host',
+  labelPlural: 'Bare Metal Hosts',
+  apiVersion: 'v1alpha1',
+  path: 'baremetalhosts',
+  apiGroup: 'metalkube.org',
+  plural: 'baremetalhosts',
+  abbr: 'BMH',
+  namespaced: true,
+  kind: 'BareMetalHost',
+  id: 'baremetalhost',
+  crd: true
```

Comment:
Nit: missing trailing comma, you can run `yarn lint` in the monorepo root to ensure there are no linting issues.

Reply:
Done